.github/workflows/rolling-pre-commit.yml

```diff
@@ -6,6 +6,11 @@ on:
     branches:
       - master
 
+concurrency:
+  # cancel previous runs of the same workflow, except for pushes on master branch
```

Comment:
This doesn't contain the exception, right?

Reply:
```suggestion
```